apps/req.c

```diff
@@ -545,6 +545,15 @@ int req_main(int argc, char **argv)
                     goto end;
                 }
             }
+
+#ifndef OPENSSL_NO_EC
+            if (pkey_type == EVP_PKEY_EC
+                    && EVP_PKEY_CTX_get_ec_keygen_bits(genctx, &newkey) <= 0) {
```

Comment:
I did wonder about submitting an alternative PR with that solution but thought it might be useful (and more consistent) if the EC key size is available. As for the release scheduling - not for me to say! :)